components/expand-collapse/expand-collapse-content.js

```diff
@@ -44,8 +46,17 @@ class ExpandCollapseContent extends LitElement {
 
 			.d2l-expand-collapse-content-container {
 				display: none;
+				opacity: 0;
 				overflow: hidden;
-				transition: height 400ms cubic-bezier(0, 0.7, 0.5, 1);
+				transition:
+					height var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function),
+					opacity var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function);
+			}
+
+			.d2l-expand-collapse-content-container[data-state="expanding"],
+			.d2l-expand-collapse-content-container[data-state="expanded"],
+			.d2l-expand-collapse-content-container[data-state="collapsing"] {
+				display: block;
 			}
```

Comment:
😆 I'm going to ask the same question here - aren't these already captured under the `:not([data-state="collapsed"])`? Or is there a state where it can be both `collapsed` and e.g. `collapsing`?

Reply:
ughhhh.... did not mean to commit this hunk sorry.